src/main/api/catalog.ts

```diff
@@ -85,17 +85,35 @@ export class CatalogApi implements ICatalogApi {
             }
         }
 
+        const isAudiobook = (item: PublicationView) =>
+            item.RDFType && /http[s]?:\/\/schema\.org\/Audiobook$/.test(item.RDFType);
+
+        const lastAdded = lastAddedPublicationViews.filter((item) => !isAudiobook(item));
+        const lastRead = lastReadPublicationViews.filter((item) => !isAudiobook(item));
+        const audiobooks = lastAddedPublicationViews.filter(isAudiobook);
+        const lastAudiobooks = lastReadPublicationViews.filter(isAudiobook);
```

Comment:
I find this code difficult to read, because of the variable names. I would like to propose:
```
lastAddedPublication
lastReadPublication
lastAddedAudioBooks
lastReadAudioBooks
```